src/recordLastNum.js

```diff
@@ -0,0 +1,18 @@
+const recordLastNum = (result) => {
```

Comment:
@grimxreaper You changed the behavior of the code when extracting this function. What are some issues with that?

Reply:
By this function do you mean `recordLastNum`? I don't have that function currently, I don't think I've had that at all in this refactor because it has been `handleDoubleEquals` in the past, so I'm not sure if you mean me having this function introduced a bug or...?

Reply:
Look carefully at the commit titled "extract recordLastNum" and try to identify what behavior changed.

Reply:
Figure it out yet? @grimxreaper

Reply:
Yes I solved the bug.

> You changed the behavior of the code when extracting this function. What are some issues with that?
 
That when it is referenced again elsewhere it will probably have to be referenced differently (hope that makes sense).

Reply:
You didn’t just move code, you changed what it did. You added operator code in where there was no operator code before. This could result in hard to catch non-obvious bugs. In a refactor we try to move code without changing behavior. Even minor changes.

Reply:
`operate` specifically

Reply:
Ahh, okay, noted!